diff -uraN etlegacy-2.82.1/src/db/db_sqlite3.c omv-etlegacy-2.82.1/src/db/db_sqlite3.c
--- etlegacy-2.82.1/src/db/db_sqlite3.c	2024-04-19 17:47:10.000000000 +0200
+++ omv-etlegacy-2.82.1/src/db/db_sqlite3.c	2024-05-16 17:51:07.165392674 +0200
@@ -158,31 +158,6 @@
 
 		if (db_mode->integer == 1)
 		{
-			// init memory table
-			result = sqlite3_open_v2("file::memory:?mode=memory&cache=shared", &db, (SQLITE_OPEN_READWRITE | SQLITE_OPEN_MEMORY | SQLITE_OPEN_SHAREDCACHE), NULL);
-
-			if (result != SQLITE_OK)
-			{
-				Com_Printf("... failed to open memory database - error: %s\n", sqlite3_errstr(result));
-				(void) sqlite3_close(db);
-				db = NULL;
-				return qfalse;
-			}
-
-			result = sqlite3_enable_shared_cache(1);
-
-			if (result != SQLITE_OK)
-			{
-				Com_Printf("... failed to share memory database - error: %s\n", sqlite3_errstr(result));
-				(void) sqlite3_close(db);
-				db = NULL;
-				return qfalse;
-			}
-			else
-			{
-				Com_Printf("... shared cache enabled\n");
-			}
-
 			// load from disk into memory
 			result = DB_LoadOrSaveDb(db, to_ospath, 0);
 
